ptd_client_server/lib/server.py

```diff
@@ -61,6 +61,7 @@
 
 # https://github.com/mlcommons/power-dev/issues/154#issuecomment-785188217
 MULTICHANNEL_DEVICES = [48, 59, 61, 77]
+DEVICE_TYPE_WT500 = 48
```

Comment:
Please add reference to https://github.com/mlcommons/power-dev/issues/220 here.

Reply:
Added.